astro/src/content/docs/customize/look-and-feel/upgrade.mdx

```diff
@@ -20,6 +20,16 @@ New versions of FusionAuth sometimes include new or updated theme templates. If
 
 This page contains notes on changes in recent versions of FusionAuth that require changes to a customized theme.
 
+## Version 1.53.3
+
+Version `1.53.3` includes a change to persist the user's SSO decision through an external identity provider workflow. If the Google IdP's `loginMethod` is configured as `UsePopup` or `UseVendorJavaScript`, existing custom advanced themes require an update to the `helpers` template to incorporate the fix for the Google IdP. You can update the template via the API or by modifying the *Helpers* template in the admin UI.
```

Comment:
Should we use similar language from the release notes? 

For example, the release notes say:

> ... end user is generally shown a checkbox named `Keep me signed in`, which indicates whether the user wishes to create an SSO session after logging in.

So here if we describe this as the `users's SSO decision` I think this equal to the user's selection for the `checkbox named 'Keep me signed in'`. 

Can we use some similar language to connect the two concepts?